docs/tutorials/data-loading/local-files.md

```diff
@@ -0,0 +1,131 @@
+Only one instance of Qlik Associative Engine is needed and it is started with all necessary data files mounted.
+Change `ACCEPT_EULA` as applicable and run:
+
+```sh
+ACCEPT_EULA=<yes/no> docker-compose up -d
+```
+
+Now install dependencies:
+
+```sh
+npm install
+```
+
+## Sample data files
+
+The examples use these files to load data from:
+
+* [airports.csv](https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.csv) -
+  Headers on first row, comma-separated data
+* [airports.xlsx](https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.xlsx) -
+  Headers on first row, MS Excel Open XML
+
+The files are mounted into the Qlik Associative Engine and made available on the file system visible from the container.
+
+## Examples
+
+### CSV file
+
+This example uses the [load-csv-file](https://github.com/qlik-oss/core-data-loading/blob/master/scripts/load-csv-file)
+load script to load data from the
+[airports.csv](https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.csv) file into Qlik Associative
+Engine.
+
+Script characteristics:
+
+* Names the resulting table `Airports`
+* Loads data from all headers (`*` notation)
+* Specifies that the file is in text format and `utf8` encoded
+* Sets the value delimiter to the `,` character
+* Uses embedded lables from the CSV file itself as field names, reading labels from the first line by default
+
+```sh
+npm start load-csv-file
+```
+
+The exepected output is a list of all airports loaded with field names as given by the headers in the CSV file. Also
+study the [code](https://github.com/qlik-oss/core-data-loading/blob/master/index.js) for invoking the load script using
+[enigma.js](https://github.com/qlik-oss/enigma.js/).
+
+### Excel file
+
+This example is similar to the previous example, but now uses the
+[load-xlsx-file](https://github.com/qlik-oss/core-data-loading/blob/master/scripts/load-xlsx-file) load script to load
+the same airport data from the
+[airports.xlsx]((https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.xlsx)) Excel file instead.
+
+Script characteristics:
+
+* Names the resulting table `Airports`
+* Loads data from all headers (`*` notation)
+* Specifies that the file is in in Excel format (`ooxml`)
+* Uses embedded lables from the CSV file itself as field names
+* Specifies that the table data shall be retrieved from the `Airports` sheet of the Excel file
+
+```sh
+npm start load-xlsx-file
+```
+
+The expected output is the list of all airports just as in the previous example.
+
+### Subset of fields
+
+This example shows how the
+[load-subset-of-fields](https://github.com/qlik-oss/core-data-loading/blob/master/scripts/load-subset-of-fields) script
+only loads a subset of the fields from the
+[airports.csv](https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.csv) file.
+
+Using the `*` notation in the `LOAD` statement is convenient when loading all data but sometimes you may want to only
```

Comment:
Think we should use fields instead of data (might be confusing with the amount of rows